Src/WitsmlExplorer.Frontend/components/Sidebar/FilterPanel.tsx

```diff
@@ -62,6 +69,17 @@ const FilterPanel = (): React.ReactElement => {
             }
             label={"Show only growing logs"}
           />
+          <Divider />
+          <SearchTextField
```

Comment:
I think the name of this element should be changed to something more generic as we do not perform search with this.

I'd also recommend adding autoComplete={"off"} attribute because the browser recommended some passwords to input for me but I cannot reproduce it unfortunately.

I think some padding would look great here too (the label touches the input value)

Reply:
Fixed except the padding. Too much for this PR, and should in that case be done for all TextFields. If it becomes a problem we should make a separate issue.